packages/fast-components-react-msft/build/create-sketch-library.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Generate the sketch library file for MSFT components.
+ *
+ * To use this script:
+ * 1. Ensure you have the development site running - from the root of this project, run `npm start` and wait until the website is built
+ * 2. run `node build/create-sketch-library.js`
+ * 3. Load the output `fast-dna-msft-design-kit.json` file into sketch using the plugin from `https://github.com/brainly/html-sketchapp/releases/`
+ * 4. Save out the file as a .sketch file to app/fast-dna-msft-design-kit.sketch
+ */
+const extractSymbolLibrary = require("@microsoft/fast-sketch-library").extractSymbolLibrary;
+const fs = require("fs");
+
+const sources = [
+    "http://localhost:7001/components/button/",
+    "http://localhost:7001/components/checkbox/",
+    "http://localhost:7001/components/divider/",
+    "http://localhost:7001/components/heading/",
+    "http://localhost:7001/components/label/",
+    "http://localhost:7001/components/toggle/",
+    "http://localhost:7001/components/typography/",
+    "http://localhost:7001/components/hypertext/",
+].map(value => {
+    return {
+        url: value,
+        selectors: "[data-sketch-symbol]"
```

Comment:
I expect we'll have additional export types.  Should we plan for this in the name?  data-exports-sketch or something?  What is a symbol in this context?

Reply:
A symbol is a sketch concept. I feel like `data-exports-sketch` doesn't really change the meaning - I feel like `data-sketch-symbol` is descriptive and appropriate - if the scope of requirements changes we can easily change this, but without a clearer idea of what it would change to I don't know what else to make it.